build/update-client-gen.sh

```diff
@@ -19,6 +19,7 @@
 set -o errexit
 set -o nounset
 set -o pipefail
+set -o xtrace
```

Comment:
Did you mean to commit this?

Reply:
Yes, I was a little weirded out seeing the other codegen run and not the client-generator.